src/Fable.Core/Fable.Core.JS.fs

```diff
@@ -1,447 +1,493 @@
+        static member is (value1: obj, value2: obj) : bool = jsNative
+        static member setPrototypeOf (o: obj, proto: obj) : Object = jsNative
+        static member getOwnPropertyDescriptor (o: obj, propertyKey: obj) : PropertyDescriptor = jsNative
+        static member defineProperty (o: obj, propertyKey: obj, attributes: PropertyDescriptor) : Object = jsNative
+
+    [<Global>]
+    type Math =
+        static member E: float = jsNative
+        static member LN10: float = jsNative
+        static member LN2: float = jsNative
+        static member LOG2E: float = jsNative
+        static member LOG10E: float = jsNative
+        static member PI: float = jsNative
+        static member SQRT1_2: float = jsNative
+        static member SQRT2: float = jsNative
+        static member abs (x: float) : float = jsNative
+        static member acos (x: float) : float = jsNative
+        static member asin (x: float) : float = jsNative
+        static member atan (x: float) : float = jsNative
+        static member atan2 (y: float, x: float) : float = jsNative
+        static member ceil (x: float) : float = jsNative
+        static member cos (x: float) : float = jsNative
+        static member exp (x: float) : float = jsNative
+        static member floor (x: float) : float = jsNative
+        static member log (x: float) : float = jsNative
+        static member max ([<ParamArray>] values: float[]) : float = jsNative
+        static member min ([<ParamArray>] values: float[]) : float = jsNative
+        static member pow (x: float, y: float) : float = jsNative
+        static member random () : float = jsNative
+        static member round (x: float) : float = jsNative
+        static member sin (x: float) : float = jsNative
+        static member sqrt (x: float) : float = jsNative
+        static member tan (x: float) : float = jsNative
+        static member clz32 (x: float) : float = jsNative
+        static member imul (x: float, y: float) : float = jsNative
+        static member sign (x: float) : float = jsNative
+        static member log10 (x: float) : float = jsNative
+        static member log2 (x: float) : float = jsNative
+        static member log1p (x: float) : float = jsNative
+        static member expm1 (x: float) : float = jsNative
+        static member cosh (x: float) : float = jsNative
+        static member sinh (x: float) : float = jsNative
+        static member tanh (x: float) : float = jsNative
+        static member acosh (x: float) : float = jsNative
+        static member asinh (x: float) : float = jsNative
+        static member atanh (x: float) : float = jsNative
+        static member hypot ([<ParamArray>] values: float[]) : float = jsNative
+        static member trunc (x: float) : float = jsNative
+        static member fround (x: float) : float = jsNative
+        static member cbrt (x: float) : float = jsNative
+
+    [<AllowNullLiteral;Global>]
+    type Date [<Emit("new Date($0)")>] private (inp: obj) =
```

Comment:
I think you don't need to use `Emit` in the constructor, Fable will detect this is a global JS object and call the constructor with `new` and the proper name thanks to the `Global` attribute. You can [test the bindings and see the generated code in the repl](https://fable.io/repl/#?code=PYBwpgdgBAYghgIwDZgHQGFgCcwChcDaAPAOJLAJxIB8AurgC4Ce4UAInA2FABQMCWAYwDWAZwBcUfhAYBKKAF5cUFVEFI4o0VEgATfCgZQuohj3lLV7TmB4BWAAyygA&html=DwCwLgtgNgfAUHUBTAhgE3gAm54ElgqbhgAOAtEgI4CuAlgG4C8A5AMID2AdmEj+QBUAnqSQtMAY268erXgA8wAenDQA3JJAoATgGcCTGmABm5ABwssOYLona6pMJl3aJTAEQB9TwAkA8gDKAt5KUHQARrpK2qgSYAB0pNocaDRxdNzxEHRc8QBWuu4wwEq29o5W2DZ2Dk4ubl6+gcGeoRFRMShx5GgcEInJqemZ2bkFRSVltfAlIKgYCMDhKUKSUCi6uh4opBRSPCg5SNpFcDi4aIyYdGgeSFDZuiDkO6Tuaxtb7vePz68TSkuDBmSmWaCE8EQKkgsDgQA&css=BYFwtgNgNAUARgewCYE8AEBvGadoGYIB2IAtAM4CWAXgKYBcaAjAGwAOAHgNwwC+MMAOjABDCoRIBjIiFGEaAJ0zZcSCmVYRhKBngg0uynAHcKSEMAaMADFYCk3XGmA0KAc1CWb9w2gBWAVzIQCjwUSWkaYgYJSJAFB1xhCDdxCjiwMmjY+J9dfRJVeRoJYKJohAh-MEJuPkExVn8QJUdWYSRVQlcGAQAmAFYisAScAmJyanomNgNHEzMLNAGrDhG0EXlXMRJEEBAEMEsh2pggA) to see what works 👍

Reply:
Oh interesting. I could have sworn I tested that in the repl and it didn't do that. Good to know!